EIPS/eip-4902.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: 4902
+title: Decentralized Autonomous Access (DAA)
+description: DAAs represent an access over digital or physical things & resources, with the ability to connect with entities that seeking to use it
+author: <Daniel @eartho-group>
+discussions-to: https://github.com/ethereum/eips/issues/4902
```

Comment:
Please open a thread on eth magicians for this proposal. This is a requirement to merge to draft.